content/docs/tasks/security/health-check/index.md

```diff
@@ -7,9 +7,11 @@ keywords: [security,health-check]
 
 You can enable Citadel's health checking feature
 to detect the failures of the Citadel CSR (Certificate Signing Request) service.
-Citadel periodically sends CSRs to its CSR service and verifies the response.
+When a failure is detected, Kubelet will be automatically start the Citadel container.
```

Comment:
will be automatically start -> will automatically restart

Reply:
Done.